### PR TITLE
Auto update h5p

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Updates webpack configuration and built files for H5P
* Adds a manual and scheduled github action to create updated PRs for the H5P JS when it is updated on their repository

## Reviewer guidance
This is an experiment to try to give dependabot regularity updating to our H5P bundled code.

If this works, we can try something similar for Bloom Reader - but that will be a little more involved, as we will have to update our fork of it as well.

The best way to test this will be to merge and run the action manually - so the main thing to look for here is that the updated files make sense, and that the action doesn't have any obvious flaws.

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by rtibbles** [comment]
I have converted this to a draft for now, we can test this action out on the test-actions repository first to avoid too much iteration here.

**Comment by rtibbles** [comment]
Did some testing on the test-actions repo to refine this, this should now be ready to go. See example PR there: https://github.com/learningequality/test-actions/pull/45

File: .github/workflows/update_h5p.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            commit_file = Path('packages/hashi/.h5p-commit-sha')
+            stored_commit = ""
+            if commit_file.exists():
+                stored_commit = commit_file.read_text().strip()
+                has_changed = stored_commit != latest_commit
+            else:
+                print("No stored commit found, treating as new")
+                has_changed = True
+            # Set outputs for GitHub Actions
+            with open(os.environ['GITHUB_OUTPUT'], 'a') as f:
```
Comment (nucleogenesis) [inline .github/workflows/update_h5p.yml:50]: I love all of this injected Python stuff, but this in particular is really neat to write to the env var as a file to speak to the GH action runner.

**Review by nucleogenesis — APPROVED** [review]
I've given this a look and see nothing blocking so should be good to merge & test

**Pull request merged**